src/UserService.Business/Commands/Avatar/AddAvatarsCommand.cs

```diff
@@ -0,0 +1,155 @@
+﻿using FluentValidation.Results;
+using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Constants;
+using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.FluentValidationExtensions;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Enums;
+using LT.DigitalOffice.Models.Broker.Requests.Image;
+using LT.DigitalOffice.Models.Broker.Responses.Image;
+using LT.DigitalOffice.UserService.Business.Commands.Avatar.Interfaces;
+using LT.DigitalOffice.UserService.Data.Interfaces;
+using LT.DigitalOffice.UserService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.UserService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.UserService.Models.Db;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.User;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.User.Avatars;
+using LT.DigitalOffice.UserService.Validation.Avatars.Interfaces;
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+
+namespace LT.DigitalOffice.UserService.Business.Commands.Avatar
+{
+  public class AddAvatarsCommand : IAddAvatarsCommand
+  {
+    private readonly IAvatarRepository _avatarRepository;
+    private readonly IUserRepository _userRepository;
+    private readonly IAccessValidator _accessValidator;
+    private readonly IAddAvatarRequestValidator _requestValidator;
+    private readonly IDbUserAvatarMapper _dbUserAvatarMapper;
+    private readonly ICreateImageDataMapper _createImageDataMapper;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IRequestClient<ICreateImagesRequest> _rcCreateImage;
+    private readonly ILogger<AddAvatarsCommand> _logger;
+
+    public AddAvatarsCommand(
+      IAvatarRepository avatarRepository,
+      IUserRepository userRepository,
+      IAccessValidator accessValidator,
+      IAddAvatarRequestValidator requestValidator,
+      IDbUserAvatarMapper dbUserAvatarMapper,
+      ICreateImageDataMapper createImageDataMapper,
+      IHttpContextAccessor httpContextAccessor,
+      IRequestClient<ICreateImagesRequest> rcCreateImage,
+      ILogger<AddAvatarsCommand> logger)
+    {
+      _avatarRepository = avatarRepository;
+      _userRepository = userRepository;
+      _accessValidator = accessValidator;
+      _requestValidator = requestValidator;
+      _dbUserAvatarMapper = dbUserAvatarMapper;
+      _createImageDataMapper = createImageDataMapper;
+      _httpContextAccessor = httpContextAccessor;
+      _rcCreateImage = rcCreateImage;
+      _logger = logger;
+    }
+
+    private List<Guid> AddImages(List<AddImageRequest> request, Guid senderId, List<string> errors)
```

Comment:
write code in one style: private methods before public (main style how i know) or private after public